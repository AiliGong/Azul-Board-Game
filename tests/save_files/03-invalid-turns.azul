BYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRULBYRUL
a
b
turn 1 B 2
turn 2 L 3
turn 3 U 3
turn 0 R 2
turn 4 R 1
turn 0 L 3
turn 0 U 3
turn 0 Y 5
turn 0 B 4
turn 5 U 1
turn 0 Y 5
turn 0 L 4
turn 0 R 2
turn 0 @ 5
turn 0 R 5
turn 0 R 0
