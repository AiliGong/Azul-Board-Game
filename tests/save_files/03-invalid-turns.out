Received invalid turn specification: turn 0 @ 5
